README.md

```diff
@@ -1,43 +1,25 @@
-# A script to make you proud
-
-This repository contains a small Python program that shows that I have learned Python in this semester.
-
-The code has been developed by Mariana Montes.
-
-## Installation and usage
-
-Clone this repository with the following git command in the console (or download the ZIP file via GitHub):
-
-```sh
-git clone git@github.com:montesmariana/intro_machine_learning_using_python
-```
-
-You can import the script as a module by adding the repository to your path in a Python script or interactive notebook and then calling `import`.
-
-```python
-import sys
-sys.path.append('/path/to/intro_machine_learning_using_python')
-import script as s
-```
-
-Check out `tutorial.md` to see an example of the different functionalities of the script!
-
-You can also run the script directly with the following code in a console:
-
-```sh
-python script.py <example.json>
-```
-
-Or in Jupyter notebook with:
-
-```python
-%run script.py <example.json>
-```
-
-In both cases `example.json` stands for the `filename` argument that the script needs. You can use [the file in this repository](example.json) or a similar file of yours. Find more information on how this script works with:
-
-```sh
-python script.py --help
-```
-
-If you run this script, you become proud of yourself.
+# What I'm planning
+
+- Start from the translation job management script created for the first assignment, but expand on it to make it useful for a translation agency rather than an independent translator.
+- Three class attributes (strings) :
+    - Translator
+    - Revisor
+    - Status
+   
+   -> The default value for "Translator" and "Revisor" is "Internal", meaning that an employee of the translation agency took up the job. If the agency assigned the job to a freelancer, the default value can be changed to their name.
```

Comment:
Yeah, you could also consider having a "Contact" class that has more information about a person and that could be retrieved from some database / json / csv... e.g. something with name, surname, languages, email, phone, availabilities... and this class can be used for all three attributes.